lib/util.js

```diff
@@ -607,7 +607,7 @@ module.exports = {
     }
 
     responseBody = this.convertToPmBodyData(contentObj[cTypeHeader], cTypeHeader, this.options.indentCharacter);
-    if (cTypeHeader === APP_JSON || cTypeHeader === APP_VND_JSON) {
+    if (cTypeHeader.startsWith('application') && cTypeHeader.endsWith('json')) {
```

Comment:
Need to abstract out the logic of checking JSON headers. We check for json somewhere else in the code too, right? Why is this check not made there?